### PR TITLE
fixing minor problem in line search construction

### PR DESCRIPTION
**Pull request by RiccardoRossi**
**Description**
fixes a wrong parameter name 

Please mark the PR with appropriate tags: 
- Fast PR

**Changelog**
Please summarize the changes in one list to generate the changelog:
fixed a wrong parameter name

### CODE REVIEW

File: kratos/solving_strategies/strategies/line_search_strategy.h [hunk 1]
```diff
@@ -562,7 +562,7 @@ class LineSearchStrategy
         const bool CalculateReactions
         )
     {
-        rDefaultSettings["max_iterations"].SetInt(MaxIterations);
+        rDefaultSettings["max_iteration"].SetInt(MaxIterations);
```
Comment (loumalouomega) [inline kratos/solving_strategies/strategies/line_search_strategy.h:565]: I think we had a discussion related with this. Which name is used in the python solvers?

Comment (RiccardoRossi) [reply kratos/solving_strategies/strategies/line_search_strategy.h:565]: withoit the s.

as it is now it fails

**Review by loumalouomega — APPROVED** [review]
OK

**Comment by RiccardoRossi** [comment]
thx

**Pull request merged**